cmd/protoc-gen-gorums/gengorums/template_client.go

```diff
@@ -0,0 +1,105 @@
+package gengorums
+
+import (
+	"google.golang.org/protobuf/compiler/protogen"
+)
+
+// gorums need to be imported in the zorums file
+var clientVariables = `
+{{$context := use "context.Context" .GenFile}}
+{{$_ := use "gorums.EnforceVersion" .GenFile}}
+{{$callOpt := use "gorums.CallOption" .GenFile}}
+`
+
+var clientConfigurationInterface = `
+{{$genFile := .GenFile}}
+{{- range configurationsServices .Services}}
+	{{$service := .GoName}}
+	// {{$service}} is the client-side Configuration API for the {{$service}} Service
+	type {{$service}}ConfigurationClient interface {
```

Comment:
```go
// {{$service}}Client is the client interface for the {{$service}} service.
type {{$service}}Client interface {
```
I feel that having the `Configuration` name be part of the interface name gives it a connotation to something we can configure.